### ISSUE TITLE
examples/glisp_1.py fails, when I try acquisition_method="prob_improvement"

### ISSUE THREAD

**Issue by ksetdekov** [bemporad/GLIS#1]
Hi!
Really interested in this code for practical purposes (optimizing day-to-day choices) of preference maximizaiton.
However part of the code in the examples does not work according to the documentation description ([https://pypi.org/project/glis/#basic-glisp](https://pypi.org/project/glis/#basic-glisp)), part about **Acquisition function**

I try to run code from examples/glisp_1.py file, replacing 
```python
prob2 = GLISp(bounds=(lb, ub), n_initial_random=n_initial_random, )
```
with 
```python
prob2 = GLISp(bounds=(lb, ub), n_initial_random=n_initial_random, acquisition_method="prob_improvement")
```

**Expected behavior** - almost the same results?

**Observed** - errors
```text
---------------------------------------------------------------------------
TypingError                               Traceback (most recent call last)
[<ipython-input-3-1efd47f17693>](https://localhost:8080/#) in <cell line: 128>()
    128 for k in range(max_prefs):
    129     pref = pref_fun(x2, xbest2)  # evaluate preference
--> 130     x2 = prob2.update(pref)
    131     xbest2 = prob2.xbest
    132 X2 = np.array(prob2.X[:-1])

5 frames
[/usr/local/lib/python3.10/dist-packages/glis/solvers.py](https://localhost:8080/#) in update(self, pref_val, feasible, satisfactory)
   1490             if self.display < 2:
   1491                 with contextlib.redirect_stdout(io.StringIO()):
-> 1492                     z, cost = pso(acquisition, self.lb, self.ub, swarmsize=self.PSOswarmsize,
   1493                                   minfunc=dF * self.PSOminfunc, maxiter=self.PSOiters)
   1494             else:

[/usr/local/lib/python3.10/dist-packages/pyswarm/pso.py](https://localhost:8080/#) in pso(func, lb, ub, ieqcons, f_ieqcons, args, kwargs, swarmsize, omega, phip, phig, maxiter, minstep, minfunc, debug)
    109 
    110         # Calculate the objective's value at the current particle's
--> 111         fp[i] = obj(p[i, :])
    112 
    113         # At the start, there may not be any feasible starting point, so just

[/usr/local/lib/python3.10/dist-packages/pyswarm/pso.py](https://localhost:8080/#) in <lambda>(x)
     72 
     73     # Check for constraint function(s) #########################################
---> 74     obj = lambda x: func(x, *args, **kwargs)
     75     if f_ieqcons is None:
     76         if not len(ieqcons):

[/usr/local/lib/python3.10/dist-packages/glis/solvers.py](https://localhost:8080/#) in acquisition(xs)
   1475                     rbf_xs = self.rbf(Xs, xs.T, self.rbf_epsil * self.theta)
   1476                     v = rbf_xs - rbf_xbest
-> 1477                     return facquisition_pref_prob_improvement(v, W, self.sepvalue) + self.constrpenalty(xs)
   1478             else:
   1479                 def acquisition(xs):

[/usr/local/lib/python3.10/dist-packages/numba/core/dispatcher.py](https://localhost:8080/#) in _compile_for_args(self, *args, **kws)
    466                 e.patch_message(msg)
    467 
--> 468             error_rewrite(e, 'typing')
    469         except errors.UnsupportedError as e:
    470             # Something unsupported is present in the user code, add help info

[/usr/local/lib/python3.10/dist-packages/numba/core/dispatcher.py](https://localhost:8080/#) in error_rewrite(e, issue_type)
    407                 raise e
    408             else:
--> 409                 raise e.with_traceback(None)
    410 
    411         argtypes = []

TypingError: Failed in nopython mode pipeline (step: nopython frontend)
NameError: name 'exp' is not defined
```

**Env**
1. Python 3.10.12 
2. glis==2.0.1
3. running in google colab

I can help write a jupyter notebook and pull request it into the package if there exist a solution.

**Comment by bemporad** [comment]
This has been fixed in version 2.0.2.

**Comment by ksetdekov** [comment]
> This has been fixed in version 2.0.2.

Thanks a lot for the update! Everything works in the example as intended and described in the paper. Thanks for a quick update.
